rsocket-core/src/main/java/io/rsocket/RSocketClient.java

```diff
@@ -103,22 +104,74 @@
         .doOnNext(this::handleIncomingFrames)
         .doOnError(errorConsumer)
         .subscribe();
+
+    connection
+        .send(sendProcessor)
+        .doOnError(
+            t -> {
+              Collection<Subscriber<Payload>> values;
```

Comment:
I find it hard to follow any sufficiently big chain of rx operators with sizable lambdas.  Can we split these up into individual methods without losing anything?